.github/workflows/deploy-personal-dev-environment.yml

```diff
@@ -63,30 +69,126 @@ jobs:
 
       - uses: ./.github/actions/setup-zsh
 
-      - name: Checkout all repos
-        run: |
-          source uhd.sh
-          uhd gh clone
-        shell: zsh {0}
+      - name: Checkout frontend repo
+        uses: actions/checkout@v4
+        with:
+          repository: UKHSA-Internal/data-dashboard-frontend
+          path: data-dashboard-frontend
```

Comment:
This allows us to checkout the application repo side by side to the current infra repo. Since the `uhd docker build` command depends on the repos being next to each other